packages/cdktf-cli/templates/csharp/.hooks.sscaff.js

```diff
@@ -42,12 +42,25 @@ exports.post = options => {
         <add key="NuGet official package source" value="https://api.nuget.org/v3/index.json" />
       </packageSources>
     </configuration>`, 'utf-8');
+  } else {
+    writeFileSync('./NuGet.Config', `<?xml version="1.0" encoding="utf-8"?>
+    <configuration>
+        <packageSources>
+            <add key="github" value="https://nuget.pkg.github.com/hashicorp/index.json" />
+        </packageSources>
+        <packageSourceCredentials>
+            <github>
+                <add key="Username" value="%GITHUB_USER%" />
+                <add key="ClearTextPassword" value="%GITHUB_TOKEN%" />
+            </github>
+        </packageSourceCredentials>
+    </configuration>`, 'utf-8');
   }
 
-  execSync(`dotnet restore`, { stdio: 'inherit' });
+  // execSync(`dotnet restore`, { stdio: 'inherit' });
```

Comment:
Why comment out the restore and synth?

Reply:
It's causing errors, since the `Nuget.Config` isn't complete. Essentially, the user has to authenticate manually and edit the config - unless you know a way to do this interactively? `synth` then errors, because the dependencies aren't resolved.

Reply:
I guess it comes down to when should the user get the error messaging. Unless they have things already setup, they'll either get an error on the init or an error when trying to do anything else. 🤷

Reply:
The reasoning is, that the help text will guide the user how to proceed without running into any error.